c2corg_ui/static/js/documentediting.js

```diff
@@ -33,7 +33,14 @@ app.documentEditingDirective = function() {
     scope: true,
     controller: 'appDocumentEditingController',
     controllerAs: 'editCtrl',
-    bindToController: true
+    bindToController: true,
+    link: function(scope, el, attrs, ctrl) {
+      // The directive executes after the dom is loaded but we have to wait until the expressions are evaluated
```

Comment:
not the document data, but it gives it a little bit of time for the ng-ifs. How do i know when the ng ifs are finished evaluating? I changed to
 `    
link: function(scope, el, attrs, ctrl) {
     ctrl.updateMaxSteps();
    }`

and it works ok, but when I set it in the controller like `this.max_steps = this.updateMaxSteps()` it never gets the correct value (at time).